verifier/src/main/resources/object-contract.java

```diff
@@ -7,6 +7,6 @@ class ObjectContract {
     public ObjectContract() {}
 }
 
-@Contract(Record.class)
+@Contract(value = Record.class, immutable = true)
 class RecordConstract {
```

Comment:
nit: May as well fix this typo
```suggestion
class RecordContract {
```